libs/designsystem/src/lib/components/calendar/calendar.component.spec.ts

```diff
@@ -383,6 +383,13 @@ describe('CalendarComponent', () => {
         expect(spectator.component.navigatedYear).toEqual(firstNavigatedYearIndex + 1);
       });
 
+      it('should emit selected year on navigation', () => {
+        const captured = captureYearSelectEvents();
+        spectator.component._changeYear(2040);
```

Comment:
This is very tightly coupled to the implementation, consider changing it up with `spectator.triggerEventHandler(DropdownComponent, 'change', '2040');`